coriolis/osmorphing/windows.py

```diff
@@ -146,9 +147,21 @@ def _dismount_disk_image(self, path):
         self._conn.exec_ps_command("Dismount-DiskImage '%s'" % path,
                                    ignore_stdout=True)
 
-    def _expand_archive(self, path, destination):
+    @utils.retry_on_error()
+    def _expand_archive(self, path, destination, overwrite=True):
         LOG.info("Expanding archive \"%(path)s\" in \"%(destination)s\"",
                  {"path": path, "destination": destination})
+
+        if self._conn.test_path(destination):
+            LOG.info("Destination folder %s already exists" % destination)
+            if overwrite:
+                if destination.endswith(":\\") or ":\\Windows" in destination:
+                    LOG.warn(
+                        "Not removing target directory, as it is either the "
```

Comment:
Do we run the risk of it asking for overwrite confirmation and hanging if we ignore this here?
If so, I suggest we just error out and impose that the plugins not directly unarchive shit in C:\

Reply:
the expand archive commands are non-interactive.

Reply:
Cool. LGTM then.